### PR TITLE
Kube deploy fixups

### PR DESCRIPTION
**Pull request by AlCutter**
- PROJECT_NAME -> PROJECT_ID
- Make Map deployment optional
- Fix name of map container

### CODE REVIEW

File: examples/deployment/kubernetes/create.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ if ! jq --help > /dev/null; then
 fi
 
 echo "Creating new Trillian deployment"
-echo "  Project name: ${PROJECT_NAME}"
+echo "  Project name: ${PROJECT_ID}"
 echo "  Cluster name: ${CLUSTER_NAME}"
 echo "  Region:       ${REGION}"
 echo "  Node Locs:    ${NODE_LOCATIONS}"
```
Comment (Martin2112) [inline examples/deployment/kubernetes/create.sh]: Add `${RUN_MAP} to the summary?

File: examples/deployment/kubernetes/deploy.sh [hunk 2]
```diff
@@ -56,15 +60,21 @@ kubectl delete configmap deploy-config || true
...
-kubectl set image deployment/trillian-logsigner-deployment trillian-log-signer=gcr.io/${PROJECT_NAME}/log_signer:${IMAGE_TAG}
-kubectl set image deployment/trillian-mapserver-deployment trillian-mapserver=gcr.io/${PROJECT_NAME}/map_server:${IMAGE_TAG}
+kubeconfigs="trillian-log-deployment.yaml trillian-log-service.yaml trillian-log-signer-deployment.yaml trillian-log-signer-service.yaml"
+if ${RUN_MAP}; then
+  kubeconfigs+=" trillian-map-deployment.yaml trillian-map-service.yaml"
+fi
+for thing in ${kubeconfigs}; do
+  echo ${thing}
+  envsubst < ${DIR}/${thing} | kubectl apply -f -
+don
```
Comment (Martin2112) [inline examples/deployment/kubernetes/deploy.sh]: Who is don?

File: examples/deployment/kubernetes/create.sh (hunk 1 above)
Comment (AlCutter) [reply examples/deployment/kubernetes/create.sh]: I figured that was more about the deploy than the creation?

File: examples/deployment/kubernetes/deploy.sh (hunk 2 above)
Comment (AlCutter) [reply examples/deployment/kubernetes/deploy.sh]: Don's done.

File: examples/deployment/kubernetes/create.sh (hunk 1 above)
Comment (Martin2112) [reply examples/deployment/kubernetes/create.sh]: Yeah maybe. Might be worth adding it somewhere that there's no map being set up but not a big deal.

**Pull request merged**